src/test/java/org/elasticsearch/rest/action/admin/indices/upgrade/UpgradeTest.java

```diff
@@ -246,6 +243,7 @@ public static void runUpgrade(HttpRequestBuilder httpClient, String index, Strin
         assertEquals(200, rsp.getStatusCode());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
While the changes to this file seem fine, I don't understand what they have to do with this PR?